lang/en/tool_objectfs.php

```diff
@@ -88,6 +88,8 @@
 $string['settings:aws:bucket_help'] = 'Amazon S3 bucket to store files in.';
 $string['settings:aws:region'] = 'region';
 $string['settings:aws:region_help'] = 'Amazon S3 API gateway region.';
+$string['settings:aws:base_url'] = 'Base URL';
+$string['settings:aws:base_url_help'] = 'Alternate url for cnames or s3 compatible endpoints';
```

Comment:
Can you please flesh this out to make it clear that it should be left blank for normal s3